heartbeat/monitors/wrappers/summarizer/plugstatestat.go

```diff
@@ -150,12 +150,13 @@ func (ssp *commonSSP) BeforeSummary(event *beat.Event) BeforeSummaryActions {
 	lastStatus := ssp.stateTracker.GetCurrentStatus(ssp.sf)
 
 	curCheckDown := ssp.js.Status == monitorstate.StatusDown
-	lastStateUp := ssp.stateTracker.GetCurrentStatus(ssp.sf) == monitorstate.StatusUp
+	lastStateUpOrEmpty := ssp.stateTracker.GetCurrentStatus(ssp.sf) == monitorstate.StatusUp ||
+		ssp.stateTracker.GetCurrentStatus(ssp.sf) == monitorstate.StatusEmpty
 	hasAttemptsRemaining := ssp.js.Attempt < ssp.js.MaxAttempts
 
 	// retry if...
 	retry := curCheckDown && // the current check is down
-		lastStateUp && // we were previously up, if we were previously down we just check once
+		lastStateUpOrEmpty && // we were previously up or had no previous state, if we were previously down we just check once
```

Comment:
Can you change this line to log Line from debug to Info 
```
logp.L().Debugf("attempt info: current(%v) == lastStatus(%v) && attempts(%d < %d)", ssp.js.Status, lastStatus, ssp.js.Attempt, ssp.js.MaxAttempts)
```

Helps with debugging on the service.